modules/swagger-core/src/test/java/io/swagger/BeanValidatorTestNG.java

```diff
@@ -0,0 +1,37 @@
+package io.swagger;
+
+import io.swagger.converter.ModelConverters;
+import io.swagger.models.Model;
+import io.swagger.models.properties.DoubleProperty;
+import io.swagger.models.properties.IntegerProperty;
+import io.swagger.models.properties.Property;
+import io.swagger.models.properties.StringProperty;
+import models.BeanValidationsModel;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+public class BeanValidatorTestNG extends Assert {
```

Comment:
Yes, but we have some java tests in scala (i.e. SimpleGenerationTest, ModelPropertyOverrideTest, etc.). And these tests will be run two times (first run by the ScalaTest and second run by the TestNG).
It's best workaround when we have two types of tests.